### ISSUE TITLE
Remove return keyword when returning single expression from a function

### ISSUE THREAD

**Issue by addityasingh** [lebab/lebab#210]
A function returning a single value can use the expression form () to return the value without using `return` keyword. These are very common scenarios while using **Redux action-creators** with **Redux thunk** or **Redux Promise**

##### From:
```javascript
const actionCreator = () => {
  return {
    type: 'ADD_COUNTER',
    payload: { count: 1}
  }
}
```

##### To:
```javascript
const actionCreator = () => ({
    type: 'ADD_COUNTER',
    payload: { count: 1}
})
```

This could also be applied to single function, single Promise or any single expressions being returned. E.g

##### From:
```javascript
const actionCreator = () => {
  return (dispatch, getState) => { console.log(k); }
}
```

##### To:
```javascript
const actionCreator = () => 
  (dispatch, getState) => { console.log(k); }
```

##### From:
```javascript
const getSinglePromise = () => {
  return new Promise((resolve, request) => {
    if (true) {
      resolve('Truthy value');
    } else {
      reject('Falsy value');
    }
  });
}
```

##### To:
```javascript
const getSinglePromise = () => 
  new Promise((resolve, request) => {
    if (true) {
      resolve('Truthy value');
    } else {
      reject('Falsy value');
    }
  });
```

**Comment by nene** [comment]
I guess you're suggesting a new transform, as this is what the `arrow` transform already does when the input is a normal non-arrow function.

Should be a pretty simple feature to add. Perhaps it even makes sense to separate this from `arrow` transform to a separate transform.

**Comment by addi90** [comment]
Yeah. But I tried and the arrow transform doesn't remove the return for expressions which I have mentioned. So I wanted this to be an additional one. I am looking into it to make this work. Will give a PR once I have it working

**Comment by nene** [comment]
Awesome!

**Comment by nene** [comment]
Solved by #269
